src/components/base/outline-alert/outline-alert.ts

```diff
@@ -0,0 +1,78 @@
+import { html, TemplateResult, CSSResultGroup } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import componentStyles from './outline-alert.css.lit';
+import { OutlineElement } from '../outline-element/outline-element';
+
+export const alertSizes = ['small', 'large'] as const;
+export type AlertSize = typeof alertSizes[number];
+
+export const alertStatusTypes = [
+  'information',
+  'warning',
+  'error',
+  'success',
+] as const;
+export type AlertStatusType = typeof alertStatusTypes[number];
+
+// This can be useful for testing.
+export interface OutlineAlertInterface extends HTMLElement {
+  statusType: AlertStatusType;
+  size: AlertSize;
+  isInteractive: Boolean;
+  shouldShowIcon: Boolean;
+}
+
+/**
+ * The Outline Alert component
+ *
+ * @element outline-alert
+ * @slot default - The alert contents
+ * @slot outline-alert--header - The header in the alert
+ */
+@customElement('outline-alert')
+export class OutlineAlert
+  extends OutlineElement
+  implements OutlineAlertInterface
+{
+  static styles: CSSResultGroup = [componentStyles];
+
+  @property({ type: String })
+  statusType: AlertStatusType = 'information';
+
+  /**
+   * This is important context for screen readers.
+   */
+  @property({ type: Boolean })
+  isInteractive = false;
+
+  @property({ type: Boolean })
+  shouldShowIcon = true;
```

Comment:
This probably needs to be `shouldHideIcon` because of the way attributes work in the DOM. The presence of an attribute means `true`, so there is no way to set a false value if the default is `true`. :(

So, all boolean attributes probably need to default to false.

Reply:
Interesting 🤯